dev/test/bulk-writer.ts

```diff
@@ -125,31 +125,40 @@ describe('BulkWriter', () => {
     const writeResults = responses.map(response => ({
       updateTime: response.updateTime,
     }));
-    const status = responses.map(response => ({code: response.code!}));
+    const status = responses.map(
+      response => ({code: response.code!} as proto.google.rpc.IStatus)
+    );
     return {
       writeResults,
       status,
     };
   }
 
+  /**
+   * Creates an instance with the mocked objects.
+   *
+   * @param checkActiveRequests Whether to check for active requests. If true,
```

Comment:
Should this be `checkActiveRequestCount`?  Or even `enforceSingleConcurrentRequest`?